content/pom.xml

```diff
@@ -122,6 +122,7 @@
                         <exclude>**/*.editorconfig</exclude>
                         <exclude>**/*.stylelintrc.yaml</exclude>
                         <exclude>**/*.eslintignore</exclude>
+						<exclude>**/*.stylelintignore</exclude>
```

Comment:
@gitrajat - this doesn't seem resolved, we use 4 spaces rather than tabs across the project for indentation.

Reply:
I have worked on it, pushing changes in a while.